third_party/2/werkzeug/wrappers.pyi

```diff
@@ -2,78 +2,85 @@
-    def path(self): ...
-    def full_path(self): ...
-    def script_root(self): ...
-    def url(self): ...
-    def base_url(self): ...
-    def url_root(self): ...
-    def host_url(self): ...
-    def host(self): ...
-    query_string = ...  # type: Any
-    method = ...  # type: Any
+    def get_data(self, cache: bool=True, as_text: bool=False, parse_form_data: bool=False) -> str: ...
+    form = ...  # type: ImmutableMultiDict
+    values = ...  # type: CombinedMultiDict
+    files = ...  # type: MultiDict
+    cookies = ...  # type: TypeConversionDict
+    headers = ...  # type: EnvironHeaders
+    path = ...  # type: unicode
+    full_path = ...  # type: unicode
+    script_root = ...  # type: unicode
+    url = ...  # type: unicode
+    base_url = ...  # type: unicode
+    url_root = ...  # type: unicode
+    host_url = ...  # type: unicode
+    host = ...  # type: unicode
+    query_string = ...  # type: str
+    method = ...  # type: str
     def access_route(self): ...
     @property
-    def remote_addr(self): ...
-    remote_user = ...  # type: Any
-    scheme = ...  # type: Any
-    is_xhr = ...  # type: Any
-    is_secure = ...  # type: Any
-    is_multithread = ...  # type: Any
-    is_multiprocess = ...  # type: Any
-    is_run_once = ...  # type: Any
+    def remote_addr(self) -> basestring: ...
+    remote_user = ...  # type: basestring
+    scheme = ...  # type: str
+    is_xhr = ...  # type: bool
+    is_secure = ...  # type: bool
+    is_multithread = ...  # type: bool
+    is_multiprocess = ...  # type: bool
+    is_run_once = ...  # type: bool
 
 class BaseResponse:
-    charset = ...  # type: Any
-    default_status = ...  # type: Any
-    default_mimetype = ...  # type: Any
-    implicit_sequence_conversion = ...  # type: Any
-    autocorrect_location_header = ...  # type: Any
-    automatically_set_content_length = ...  # type: Any
-    headers = ...  # type: Any
-    status_code = ...  # type: Any
-    status = ...  # type: Any
-    direct_passthrough = ...  # type: Any
-    response = ...  # type: Any
-    def __init__(self, response=None, status=None, headers=None, mimetype=None, content_type=None, direct_passthrough=False): ...
+    charset = ...  # type: basestring
+    default_status = ...  # type: int
+    default_mimetype = ...  # type: basestring
+    implicit_sequence_conversion = ...  # type: bool
+    autocorrect_location_header = ...  # type: bool
+    automatically_set_content_length = ...  # type: bool
+    headers = ...  # type: Headers
+    status_code = ...  # type: int
+    status = ...  # type: str
+    direct_passthrough = ...  # type: bool
+    response = ...  # type: Iterable[str]
+    def __init__(self, response: Union[Iterable[str], str]=None, status=Union[basestring, int], headers: Union[Headers, Mapping[basestring, basestring], Sequence[Tuple[basestring, basestring]]]=None, mimetype: basestring=None, content_type: basestring=None, direct_passthrough: bool=False): ...
```

Comment:
This must have `-> None` (see the Travis failure). I guess it didn't fail before because the function didn't have any type annotations.